watson_developer_cloud/natural_language_classifier_v1.py

```diff
@@ -11,59 +13,488 @@
+               Bluemix, the credentials will be automatically loaded from the
+               `VCAP_SERVICES` environment variable.
+
         """
-        params = {'language': language, 'name': name}
-        return self.request(method='POST', url='/v1/classifiers',
-                            accept_json=True,
-                            files=[('training_metadata',
-                                    ('training.json', json.dumps(params))),
-                                   ('training_data', training_data)])
 
-    def list(self):
-        return self.request(method='GET', url='/v1/classifiers',
-                            accept_json=True)
+        WatsonDeveloperCloudService.__init__(
+            self,
+            vcap_services_name='',
+            url=url,
+            username=username,
+            password=password,
+            use_vcap_services=True)
 
-    def status(self, classifier_id):
-        classifier_id = self.unpack_id(classifier_id, 'classifier_id')
-        return self.request(method='GET',
-                            url='/v1/classifiers/{0}'.format(classifier_id),
-                            accept_json=True)
+    #########################
+    # naturallanguageclassifier
+    #########################
 
     def classify(self, classifier_id, text):
-        classifier_id = self.unpack_id(classifier_id, 'classifier_id')
-        return self.request(method='POST',
-                            url='/v1/classifiers/{0}/classify'.format(
-                                classifier_id), accept_json=True,
-                            json={'text': text})
-
-    def remove(self, classifier_id):
-        classifier_id = self.unpack_id(classifier_id, 'classifier_id')
-        return self.request(method='DELETE',
-                            url='/v1/classifiers/{0}'.format(classifier_id),
-                            accept_json=True)
+        """
+        Returns label information for the input.
+
+        The status must be `Available` before you can use the classifier to classify text.
+        Use `Get information about a classifier` to retrieve the status.
+
+        :param str classifier_id: Classifier ID to use.
+        :param str text: The submitted phrase.
+        :return: A `Classification` object
+        :rtype: natural_language_classifier_v1.Classification
+        """
+        if classifier_id is None:
+            raise WatsonInvalidArgument('classifier_id must be provided')
+        if text is None:
+            raise WatsonInvalidArgument('text must be provided')
+        data = json.dumps({'text': text})
+        headers = {'content-type': 'application/json'}
+        response = self.request(
+            method='POST',
+            url='/v1/classifiers/{0}/classify'.format(classifier_id),
+            headers=headers,
+            data=data,
+            accept_json=True)
+        return Classification._from_dict(response)
+
+    def classify_get(self, classifier_id, text):
```

Comment:
???? 🤔  why don't we leave it as `classify()`

Reply:
Thanks for catching that.  We had an annotation out of place. I will fix.